src/modules/products/controllers/product.controller.ts

```diff
@@ -21,23 +21,20 @@ export class ProductController {
 
   @Get()
   public async findAllProducts(
-    @Query('category', new DefaultValuePipe('All Category')) category: string,
+    // @Query('category', new DefaultValuePipe('All Category')) category: string,
     @Query('per_page', new DefaultValuePipe(0), ParseIntPipe) perPage: number,
     @Query('page', new DefaultValuePipe(1), ParseIntPipe) page: number,
     @Query('name', new DefaultValuePipe('')) name: string,
   ): Promise<Product[]> {
-    return await this.productService.findAllProducts(category, perPage, page, name);
+    return await this.productService.findAllProducts(perPage, page, name);
```

Comment:
remove `await`, it does not make any sense. `return await` only makes some sense if you cover it by `try..catch` and expecting an error

Reply:
I've took off all async/await from controllers, this statements really don't matter at all